tests/unit/test_gapjunctions.py

```diff
@@ -0,0 +1,211 @@
+import re
+from pathlib import Path
+
+import numpy as np
+import numpy.testing as npt
+import pytest
+
+from tests.utils import check_signal_peaks
+
+from .conftest import RINGTEST_DIR
+from neurodamus import Neurodamus
+from neurodamus.connection_manager import SynapseRuleManager
+from neurodamus.core.configuration import SimConfig
+from neurodamus.gap_junction import GapJunctionManager
+
+
+@pytest.mark.parametrize(
+    "create_tmp_simulation_config_file",
+    [
+        {
+            "simconfig_fixture": "ringtest_baseconfig",
+            "extra_config": {
+                "network": str(RINGTEST_DIR / "circuit_config_gj.json"),
+                "node_set": "ABC",
+                "target_simulator": "NEURON",
+                "inputs": {
+                    "Stimulus": {
+                        "module": "pulse",
+                        "input_type": "current_clamp",
+                        "delay": 5,
+                        "duration": 50,
+                        "node_set": "RingC",
+                        "amp_start": 10,
+                        "width": 1,
+                        "frequency": 50,
+                    }
+                }
+            }
+        }
+    ],
+    indirect=True,
+)
+def test_gapjunctions(create_tmp_simulation_config_file):
```

Comment:
this test has the same name of the file. I find it a little generic. May you either improve the name or split the test in smaller modules?

Reply:
PS: may you add a docstring?

Reply:
Done!